src/components/Themes.tsx

```diff
@@ -0,0 +1,79 @@
+import { PaletteMode, ThemeOptions } from "@mui/material"; // TypeScript Types
+// An interface that defines object parameters
+interface ThemeColor {
+    default: string;
+    paper: string;
+    text: string;
+}
+
+//Toggle the color scheme between light and dark mode
+export const baseTheme = (mode: PaletteMode) => {
+    let themeColor: ThemeColor;
+    if (mode === "light") {
+        themeColor = {
+            default: "#F3FFDE",
+            paper: "#F3FFDE",
+            text: "#191919",
+        };
+    } else {
+        // Defind the dark theme color
+        themeColor = {
+            default: "#232726",
+            paper: "#232726",
+            text: "#F3FFDE",
+        };
+    }
+    // allow configuration using `theme`
+    const theme: ThemeOptions = {
+        typography: {
+            fontFamily: "monospace, sans-serif",
+            button: {
+                textTransform: "capitalize",
+            },
+        },
+        zIndex: {
```

Comment:
Is this used anywhere? typically MUI modals already handle zIndex overlays effectively